lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -1357,14 +1367,16 @@ def sanitize_tag(tag)
     end
 
     # For every original_label => new_label pair in the label_map, delete the
-    # original_label from the record if it exists, and extract the value to form
-    # a map with the new_label as the key.
-    def fields_to_labels(record, label_map)
```

Comment:
Renamed this function to make it more generic. Also make it obvious what is actually being done.